### DISCUSSION TITLE
NotImplementedError: Cannot convert a symbolic Tensor to a numpy array

### DISCUSSION THREAD

**Discussion by Talioll** [teamalgoritma/community#493] [Q&A] [answered]
Selamat siang, 

Sy sedang mengerjakan capstone text mining dan menggunakan model LSTM. 

Saat mencoba mdoel pertama sesuai dengan arahan pada blog algo https://algotech.netlify.app/blog/text-lstm/ proses pembuatan model mengalami error (foto terlampir).

Mohon untuk dibantu penyelesaiannya, terima kasih :)

![error lstm](https://user-images.githubusercontent.com/112487401/208231776-e8e6662f-4ddd-4ea2-8262-0c2a80d1cd18.jpg)

**Comment by kevbow** [comment, marked as answer]
Dear @Talioll,

Berdasarkan pencarian yang kami lakukan, error terserbut muncul karena adanya masalah versi dari library dalam environment yang Anda gunakan. Sebelum dilakukan downgrade versi beberapa library tersebut, bisa coba diperlihatkan versi numpy dan tensorflow yang Anda gunakan? Versi ini dapat diakses dengan melakukan beberapa langkah berikut:
1. Buka Anaconda Prompt
2. Ketik `conda activate r-tensorflow` lalu enter (jika menggunakan library r-tensorflow yang dibuat ketika kelas NN, jika tidak silakan ganti namanya sesuai virtual environment Anda.
3. Ketik `conda list` lalu cari versi dari tensorflow dan numpy yang muncul.

Demikian yang dapat kami sampaikan untuk saat ini, silakan reply dengan foto ataupun menuliskan versi yang sudah dilihat. Terima kasih

**Reply by Talioll** [reply]
Dear @kevbow,

Untuk library yg digunakan sy menggunakan sesuai yg digunakan untuk kelas NN, berikut foto list version  yg ada pada env r-tensorflow

![tensorflow vs](https://user-images.githubusercontent.com/112487401/210198218-f2796e69-a075-48e4-9dfa-f94b79a63624.jpg)
